testing/test_config.py

```diff
@@ -1240,6 +1240,50 @@ def pytest_addoption(parser):
     assert result.ret == ExitCode.USAGE_ERROR
 
     result = testdir.runpytest("--version")
+    result.stderr.fnmatch_lines(["pytest {}".format(pytest.__version__)])
+    assert result.ret == ExitCode.USAGE_ERROR
+
+
+def test_help_and_version_verbose_after_argument_error(testdir):
```

Comment:
Thanks @debugduck!

This is not producing the output because there's special handling when there's an usage error:

https://github.com/pytest-dev/pytest/blob/5a6296a2d73d32010d4cddc4dd6cf6a289c01d9d/src/_pytest/config/__init__.py#L871-L893

When that happens, it is best to just stick to showing the minimal version information. 👍 

So while your instincts were correct in adding this test, I think we can just remove it. 

Thanks!

Reply:
Gotcha!

Okay, I removed it. I believe I resolved the git metadata and I also added the documentation for the change as well.

Thanks! :)